test/unit/lifecycle/properties.js

```diff
@@ -25,7 +26,11 @@ describe('lifecycle/property', () => {
   });
 
   describe('props declared as attributes with ES2015 classes are linked', () => {
+    const skip = !classStaticsInheritance();
```

Comment:
You can just check if `Object.setPrototypeOf` exists but you're probably right in doing this as it's more descriptive.